ncdr/static/css/styles.scss

```diff
@@ -657,3 +657,21 @@ hr {
     width: 250px;
   }
 }
+
+// Version upload form
+form.upload {
+  .form-group {
+    align-items: baseline;
+    display: flex;
+    margin-bottom: 0;
+  }
+
+  input[type="file"] {
+    display: flex;
```

Comment:
Have we checked on IE? Browser support for flex is not 100% but I'm not sure how limitted.

Reply:
Sort of? https://caniuse.com/#feat=flexbox

Do you have an easy way to test this?